packages/redux/src/raffles/actions/factories/fetchRaffleParticipationsFactory.ts

```diff
@@ -0,0 +1,43 @@
+import * as actionTypes from '@farfetch/blackout-redux/src/raffles/actionTypes';
```

Comment:
This should be a relative import since it is from the same package.

Reply:
done